sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyExportEncryptionAlgorithm.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Security.KeyVault.Keys
+{
+    /// <summary>
+    /// The encryption algorithm to use to protected the exported key material.
+    /// </summary>
+    public readonly struct KeyExportEncryptionAlgorithm : IEquatable<KeyExportEncryptionAlgorithm>
+    {
+        internal const string CkmRsaAesKeyWrapValue = "CKM_RSA_AES_KEY_WRAP";
+        internal const string RsaAesKeyWrap256Value = "RSA_AES_KEY_WRAP_256";
+        internal const string RsaAesKeyWrap384Value = "RSA_AES_KEY_WRAP_384";
+
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="KeyExportEncryptionAlgorithm"/> structure.
+        /// </summary>
+        /// <param name="value">The string value of the instance.</param>
+        public KeyExportEncryptionAlgorithm(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
```

Comment:
`Argument.AssertNotNull`?

Reply:
Fair. I had copied it from some older code that predated the helpers and didn't think about it.